vaadin-simple-grid-filter-demo/src/main/java/software/xdev/vaadin/ui/MainView.java

```diff
@@ -85,7 +86,7 @@ private void initUI()
 			.withDatePickerI18n(datePickerI18n)
 			.withDateTimePickerLocale(Locale.GERMANY)
 			.withFilterButtonText("Add filter")
-			.withUrlParameters("filter1")
+			// .withUrlParameters("filter1")
```

Comment:
Commented out code should be removed or kept

Reply:
Fixed.